source/network/palos-upgrade.html.md.erb

```diff
@@ -70,15 +70,16 @@ e.g `v9.1.0-h3 -> 9.1.11-h3 before v10.0.0`
 * Verify system is up and running, check cpu, memory, check firewall's system resources, if high wait until it settles, should not take long
 * Download if not already and install next major release i.e. `v10.0.7` or `v10.1.x`
 * Verify system is up and running, check cpu, memory etc
-* ⚠️ Note: Prod update needs a `CR` raised and done out of office hours (ideally 8pm or after) as each vm will need to be rebooted to apply the new software version, comms must be sent out to platform-operations & cloud-native-announce prior to work being carried out.
+* ⚠️ Note: Prod update needs a `CR` raised and done out of office hours (ideally 8pm or after) as each vm will need to be rebooted to apply the new software version. Comms must be sent out to platform-operations & cloud-native-announce prior to work being carried out and advise CGI (neil.green@cgi.com) as they have an app associated to one palo fw.
+* Start with passive prod vm-1 and once that has been updated, check the [Prod LB](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/0978315c-75fe-4ada-9d11-1eb5e0e0b214/resourceGroups/hmcts-hub-prod-int/providers/Microsoft.Network/loadBalancers/hmcts-hub-prod-int-palo-lb/overview) monitoring->insights is showing the vm as green before starting work on the other prod vm.
```

Comment:
Is it worth checking logs that the restarted Palo is also getting traffic?

Reply:
good call - i've added a comments to advise as such - thanks.